modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecCommandRoutes.scala

```diff
@@ -57,5 +59,25 @@ class SeqexecCommandRoutes(auth: AuthenticationService) extends BooPicklers {
 
     case req @ GET  -> Root  / obsId / "state" =>
       Ok(toSequenceConfig("state", commands.state(obsId)))
+
+    // New SeqexecEngine
+
+    // TODO: Add obsId parameter
+    case req @ POST -> Root / "start" =>
+      // TODO: Get rid of `.toString` How do we want to represent input results
+      // now?
+      Ok(SeqexecEngine.start(q).map(_.toString))
```

Comment:
Does this return a process or a `Task`? In case we want to return a non-String we probably need to define an encoder compatible with http4s